docs/platforms/react-native/user-feedback/index.mdx

```diff
@@ -1,9 +1,69 @@
 ---
 title: Set Up User Feedback
-description: "Learn more about collecting user feedback when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."
+description: "Learn how to enable User Feedback in your app."
 sidebar_order: 6000
 ---
 
+The User Feedback feature allows you to collect user feedback from anywhere inside your application at any time, without needing an error event to occur first.
+
+Note that if you're using a self-hosted Sentry instance, you'll need to be on version 24.4.2 or higher in order to use the full functionality of the User Feedback feature. Lower versions may have limited functionality.
+
+## User Feedback Widget
+
+The user feedback widget allows users to submit feedback from anywhere inside your application.
+
+To collect user feedback from inside your application use the `showFeedbackWidget` method.
+
+```javascript
+import * as Sentry from "@sentry/react-native";
+
```

Comment:
Let's also hight light that the Root component has to be wrapped with `Sentry.wrap` for this to work.